src/main/util/consts.py

```diff
@@ -91,10 +91,14 @@ class TASK(Enum):
     MAX_DIGIT = 'max_digit'
 
     @classmethod
-    def tasks(cls):
+    def tasks_values(cls):
         return [cls.PIES.value, cls.MAX_3.value, cls.ZERO.value,
```

Comment:
Maybe is it better to use `__members__` function? Is it right that we want to get all values from TASK? It is better to use `[member.value for _, member in TASK.__members__.items()]` and we may not change  the code after adding a new element in the TASK